server/tools/attestation-service-ctl/src/attestation.rs

```diff
@@ -0,0 +1,24 @@
+use anyhow::{anyhow, Result};
+use std::fs;
+use std::path::Path;
+
+use crate::attestation_api::attestation_service_client::AttestationServiceClient;
+use crate::attestation_api::{AttestationRequest, AttestationResponse};
+
+pub async fn attestation_cmd(evidence_path: &Path, socket: &str) -> Result<()> {
+    let evidence =
+        fs::read_to_string(evidence_path).map_err(|e| anyhow!("Read evidence error: {:?}", e))?;
+
+    let request = AttestationRequest {
+        evidence: evidence.into_bytes(),
+        user: None,
+    };
+
+    let mut client = AttestationServiceClient::connect(format!("http://{}", socket)).await?;
```

Comment:
What if the specified address is a non-local one? Does this have to be "http" or can it be "https"?

Reply:
This can connect to Attestation-Server deployed locally or remotely, and both "http" and "https" is supported.

Reply:
So maybe the default should be `"http://127.0.0.1:3001"` and the user should specify the protocol (URL scheme) as well when using the `--addr http://my-service.com:12345` option.
(and then we don't need the `format!()` call here)